apis/launchers.md

```diff
@@ -1,5 +1,5 @@
 ---
-description: 'Adding Badges, Progress Bars, and Quick Lists'
+description: 'Adding Badges, Progress Bars, and launching Actions'
```

Comment:
This nomenclature was changed back in the #98 but the page description still uses the old naming.